guides/doc-Provisioning_Guide/topics/Virt-Libvirt.adoc

```diff
@@ -190,7 +204,7 @@ We can predefine certain hardware settings for KVM-based virtual machines by add
 
 .Procedure
 
-To add Red Hat Virtualization details to a compute profile, complete the following steps:
+To add KVM details to a compute profile, complete the following steps:
```

Comment:
Rahul added a attribute for these types of substitutions.

Reply:
Yes, but apparently someone was replacing word "Red Hat Virtualization" with KVM in this sentence and forgot this one. This part is dedicated to KVM (libvirt) and RHV is simply incorrect here. We don't have a variable for KVM, because it's still KVM downstream.